### DISCUSSION TITLE
Welcome to einvoice Discussions!

### DISCUSSION THREAD

**Discussion by mokth** [mokth/einvoice#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mirai1337** [comment]
Hi mokth! Thanks for the hardwork, just wondering do you have any sample on XML with foreign currency? I did tried with LHDN sample but unable to get the results
